tinygrad/codegen/uops.py

```diff
@@ -328,23 +328,15 @@ def graph_dedup(self, sink):
     while len(unprocessed_nodes):
       n = unprocessed_nodes.pop(0)
       if n in all_nodes: continue
-      all_nodes[n] = None
+      all_nodes[n], early_in_degree[n] = None, 0
       for x in n.vin:
         early_in_degree[n] += 1
         children[x].append(n)
       unprocessed_nodes += list(n.vin)
-    early_queue = [x for x in all_nodes if early_in_degree[x] == 0]
-    replace_nodes: Dict[UOp, UOp] = {}
-    while len(early_queue):
-      n = early_queue.pop(0)
-      if n in replace_nodes: continue
```

Comment:
huh, I think this does nothing -
green https://github.com/tinygrad/tinygrad/commit/64eecd27e7f65ac3ff91e0ee0981e10592c3d8b4 in CI https://github.com/tinygrad/tinygrad/actions/runs/9554614732